code/api/src/modules/user/resolvers.js

```diff
@@ -64,6 +65,28 @@ export async function login(parentValue, { email, password }) {
   }
 }
 
+// Update
+export async function update(parentValue, { id, name, email, street, city, state, zip, bio, image }, { auth }) {
+  console.log(auth.user.id)
+  if(auth.user && auth.user.id > 0) {
+    return await models.User.update(
+      {
+        name,
+        email,
+        bio,
+        image,
+        street,
+        city,
+        state,
+        zip
+      },
+      {where: { id } }
```

Comment:
So fascinating to me that id has to be in an object wrapper for the sequelize where clause. Something to keep in mind when we jump into order history.